gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,235 @@
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  Any <NxCode>className</NxCode> attributes passed in on
+                  the <NxCode>NxDrawer</NxCode> will be added to
+                  the dialog element.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>onCancel</NxTable.Cell>
+                <NxTable.Cell>Function ((Event) =&gt; void)</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  The callback function passed into <NxCode>NxDrawer</NxCode> that gets called when the user
+                  closes the dialog (when the user clicks outside the drawer, when the user clicks the close button,
+                  and when the user presses the Esc key).
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>variant</NxTable.Cell>
+                <NxTable.Cell>"narrow" | "normal"</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>"normal"</NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies a style variant for the drawer. Currently, variants only differ in width.
+                  "normal" drawers are 520px wide, and "narrow" drawers are 348px wide.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>headerTitle</NxTable.Cell>
+                <NxTable.Cell>ReactNode | string</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies the header title content placed at the top, next to the close button.
+                  The text is truncated when it overflows.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>headerSubtitle</NxTable.Cell>
+                <NxTable.Cell>ReactNode | string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies the header subtitle content placed under the header title.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>headerParagraph</NxTable.Cell>
+                <NxTable.Cell>ReactNode | string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies the header paragraph content placed under the subtitle if the
+                  subtitle is specified, if not, it will be placed under the header title.
```

Comment:
Maybe change this to:
"This prop sets the header paragraph's content under the subtitle if the subtitle is specified. If its not specified, it will be placed under the header title."

Reply:
> Maybe change this to:
> "This prop sets the header paragraph's content under the subtitle if the subtitle is specified. If its not specified, it will be placed under the header title."

Fixed here: 139ae1d5bb5f3141dd1154501a7dff94e2a046d3